server/fleet/hosts.go

```diff
@@ -392,7 +392,10 @@ func PlatformFromHost(hostPlatform string) string {
 		return "linux"
 	case hostPlatform == "darwin", hostPlatform == "windows",
 		// Some customers have custom agents that support ChromeOS
-		hostPlatform == "CrOS":
+		// TODO remove this once that customer migrates to Fleetd for Chrome
+		hostPlatform == "CrOS",
```

Comment:
remove?

Reply:
nvm I see the `,` from above..

Reply:
It should stay there for now until we help that customer migrate. Thanks for looking out!